README.quickstart.cloud9.md

```diff
@@ -75,25 +75,27 @@ The Toolkit provides commands (prefixed with `AWS:`) to the AWS Cloud9 _Go to An
+| AWS Command                               | Description                                                                                                                                                                                                                                |
+| :---------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `AWS: About Toolkit`                      | Displays information about the AWS Toolkit.                                                                                                                                                                                                |
+| `AWS: Add SAM Debug Configuration`        | Creates an `aws-sam` Debug Configuration from a function in the given source file                                                                                                                                                          |
+| `AWS: Connect to AWS`                     | Connects the Toolkit to an AWS account.                                                                                                                                                                                                    |
+| `AWS: Create a new Issue on Github`       | Opens the AWS Toolkit's [New Issue page on Github](https://github.com/aws/aws-toolkit-vscode/issues/new/choose).                                                                                                                           |
+| `AWS: Create Credentials Profile`         | Creates an AWS credentials profile.                                                                                                                                                                                                        |
+| `AWS: Create Lambda SAM Application`      | Generates code files for a new AWS serverless Lambda application. For more information, see [Creating a Serverless Application](https://docs.aws.amazon.com/cloud9/latest/user-guide/latest/user-guide/create-sam.html) in the user guide. |
+| `AWS: Create new CloudFormation Template` | Creates a new boilerplate Cloudformation Template                                                                                                                                                                                          |
+| `AWS: Create new SAM Template`            | Creates a new boilerplate SAM Template                                                                                                                                                                                                     |
```

Comment:
"boilerplate" is colloquial and slightly negative, is there a more conventional word? Perhaps "default", or we don't need a word at all.

```suggestion
| `AWS: Create new SAM Template`            | Creates a new SAM Template                                                                                                                                                                                                     |
```